README.md

```diff
@@ -6,3 +6,10 @@ Libero pattern library
 The build process uses a Node.js container image to build all assets, and copy them out of the container into `export/`.
 
 `export/` can then be packaged to be released on Github, or reused elsewhere.
+
+Getting help
+------------
+
+- Report a bug or request a feature on [GitHub](https://github.com/libero/libero/issues/new/choose).
```

Comment:
There's only one template there at the moment, but it's the right link

Reply:
(I'm using this format elsewhere, eg https://github.com/libero/content-api-bundle. It'd be good to be consistent.)